index.js

```diff
@@ -102,6 +102,43 @@
 //.
 //. `Number` is the sole inhabitant of the TypeRep Number type.
 //.
+//. ## Type checking
+//.
+//. Sanctuary functions are defined via [sanctuary-def][] to provide run-time
+//. type checking. This is tremendously useful during development: type errors
+//. are reported immediately, avoiding circuitous stack traces (at best) and
+//. silent failures due to type coercion (at worst). For example:
+//.
+//. ```javascript
+//. S.inc('XXX');
+//. // ! TypeError: ‘inc’ expected a value of type FiniteNumber as its first argument; received "XXX"
+//. ```
+//.
+//. Compare this to the behaviour of Ramda's unchecked equivalent:
+//.
+//. ```javascript
+//. R.inc('XXX');
+//. // => '1XXX'
+//. ```
+//.
+//. There is a performance cost to run-time type checking. One may wish to
+//. disable type checking in certain contexts to avoid paying this cost.
+//. There are actually two versions of the Sanctuary module: one with type
+//. checking; one without. The latter is accessible via the `unchecked`
+//. property of the former.
+//.
+//. When used correctly, `S.unchecked.<name>` will behave just like `S.<name>`.
+//. When used *incorrectly*, its behaviour is unspecified.
```

Comment:
What I'm trying to say is that `S.unchecked.inc(42)` is equivalent to `S.inc(42)`, but the behaviour of `S.unchecked.inc('XXX')` is unspecified (whereas the behaviour of `S.inc('XXX')` is covered by the test suite).

Can you think of a better way to phrase this?